scripts/core/device.yml

```
@@ -383,6 +383,8 @@ etors:
       desc: "[$x_bool_t] Return true if the device supports enqueing commands to read and write pipes from the host."
     - name: MAX_REGISTERS_PER_WORK_GROUP
       desc: "[uint32_t] The maximum number of registers available per block."
+    - name: DEVICE_IP_VERSION
```

The generator actually prefixed `UR_DEVICE_INFO_` to the enumerator name so this ends up becoming `UR_DEVICE_INFO_DEVICE_IP_VERSION` which is an unnecessary repetition of `DEVICE`.

```suggestion
    - name: IP_VERSION
```

---

Applied in https://github.com/oneapi-src/unified-runtime/pull/573/commits/53eb2104392c07fbdfa3a6425592164a977bdf1b